content/php-client/resources/products/product-media-files.md

```diff
@@ -33,6 +33,13 @@ $mediaFile = $client->getProductMediaFileApi()->download('code/example');
 file_put_contents('/tmp/ziggy.jpg', $product->getContents());
 ```
 
+From v4 of the PHP client, the response is returned instead of the content. It allows to get the filename and the Mime type from the response.
```

Comment:
```suggestion
From the v4 of the PHP client, the response is returned instead of the content. It allows getting the filename and the MIME type from the response.
```